wgsl/index.bs

```diff
@@ -2861,16 +2861,15 @@ the constant is <dfn noexport>pipeline-overridable</dfn>. In this case:
     and must be an integer value between 0 and 65535.
   * Pipeline constant IDs must be unique within the [SHORTNAME] program: Two module constants
     must not use the same pipeline constant ID.
-  * The application can specify its own value for the name at pipeline-creation time.
+  * The application can specify its own value for the constant at pipeline-creation time.
     The pipeline creation API accepts a mapping from overridable constant to a
     value of the constant's type.
-    The constant is specified by the pipeline constant ID if it was specified
-    or by the constant's identifier otherwise.
-    If the mapping has an entry for the constant, the value in the mapping is used.
-    Otherwise, the initializer expression must be present, and its value is used.
-
-Issue(dneto): What happens if the application supplies a constant ID that is not in the program?
-Proposal: pipeline creation fails with an error.
+    The constant is identified by a <dfn export>pipeline-overridable constant identifier string</dfn>,
+    which is the base-10 representation of the [=pipeline constant ID=] if specified, and otherwise
+    the declared [=name=] of the constant.
+  * The <dfn export>pipeline-overridable constant has a default value</dfn> if
+    it was declared with an initializer expression.
+    If it doesn't, a value must be provided at pipeline-creation time.
```

Comment:
time needs to be changed: "if it doesn't" -> "if it wasn't"

Reply:
The "doesn't" is supposed to go with "has a default value".

Reply:
mmm, sorry, might be my misunderstanding of the language. What it reads now: "Statement X if condition Y. If condition Z, statement W". Seems pretty clear that "if condition Y" and "if condition Z" should be the same time, doesn't it?

Reply:
Ah, I think the tense mismatch is really between "has a default value" and "was declared". It's not strictly wrong but there's no reason that it should be this way.